### ISSUE TITLE
Unused variables in the kernel

### ISSUE THREAD

**Issue by dagamayank** [ROCmSoftwarePlatform/MIOpenGEMM#11]
Error Building OpenCL Program in BuildProgram()
/tmp/AMD_7175_116/t_7175_118.cl:231:15: error: unused variable 'b_offset_perp_unroll' [-Werror,-Wunused-variable]
  const TINTB b_offset_perp_unroll = MICRO_B_TILE_PERP_UNROLL * perp_unroll_b_load_id;
              ^
/tmp/AMD_7175_116/t_7175_118.cl:274:15: error: unused variable 'a_offset_perp_unroll' [-Werror,-Wunused-variable]
  const TINTA a_offset_perp_unroll = MICRO_A_TILE_PERP_UNROLL * perp_unroll_a_load_id;

**Comment by newling** [comment]
What is your setup to enable `-Wunused-variable`? I get an invalid compiler option error


```
std::string build_options("-Werror -Wunused-variable");
```
```
Error -66 : CL_INVALID_COMPILER_OPTIONS.
```
```
uname -r
4.11.0-kfd-compute-rocm-rel-1.6-148
```

**Comment by newling** [comment]
"-Werror  -Wf,-Wno-unused-variable"

**Comment by dagamayank** [comment]
@newling Has this issue been fixed? If yes, can you please point me the commit which fixes this?

**Comment by newling** [comment]
I have not yet investigated this.

**Comment by newling** [comment]
Unused variable warning hopefully resolved with commit 132bf71758585f2cb42e6ccec6244c5db4bf4416.
